api_tests/preprints/views/test_preprint_list.py

```diff
@@ -509,9 +511,16 @@ def test_file_not_osfstorage(self):
         assert res.json['errors'][0]['detail'] == 'This file is not a valid primary file for this preprint.'
 
     def test_preprint_contributor_signal_sent_on_creation(self):
-        # Signal sent but bails out early without sending email
+        project_owner = AuthUserFactory()
+        other_user_project = ProjectFactory(creator=project_owner)
+        other_user_project.add_contributor(self.user, permissions=permissions.WRITE, visible=True, save=True)
+        other_user_project.remove_contributor(self.user, auth=Auth(project_owner))
```

Comment:
In addition, I don't understand why adding a supplementary_project is the fix. I think we are testing signal sent when preprint is created, instead of when user is added as a contributor to a project.